connection_pool/connection_pool.go

```diff
@@ -1028,3 +1028,47 @@ func newErrorFuture(err error) *tarantool.Future {
 	fut.SetError(err)
 	return fut
 }
+
+// ServerProtocolVersion returns protocol version supported by
+// Tarantool server for a connection selected by userMode from connPool.
+// Since 1.10.0
+func (connPool *ConnectionPool) ServerProtocolVersion(userMode Mode) (tarantool.ProtocolVersion, error) {
+	conn, err := connPool.getNextConnection(userMode)
+	if err != nil {
+		return tarantool.ProtocolVersionUnsupported, err
+	}
+	return conn.ServerProtocolVersion(), nil
+}
```

Comment:
1. We have multi submodule too.
2. I don't understand how to use it. Well, I found out that one of the connections supports something, so what of it?

It looks more useful in this context:
https://github.com/tarantool/go-tarantool/pull/226#issuecomment-1312781708

Reply:
Implemented in new version